examples/offline/d4rl_cql.py

```diff
@@ -73,6 +74,33 @@ def get_args():
     return parser.parse_args()
 
 
+def add_returns(buffer: ReplayBuffer, gamma: float = 0.99) -> ReplayBuffer:
```

Comment:
There is a function in `policy.base` called `_gae_return` which is used for calculating returns in `BasePolicy.compute_episodic_return`. Why did you have to add your own one? Is it different from the already existing functionality?

Reply:
`_gae_return` doesn't compute returns but rather advantages, and `BasePolicy.compute_episodic_return` acts on `Batch` rather than the `ReplayBuffer`, it can be used in 'BasePolicy.preprocess_fn' however having to preprocess each batch separately slows the execution considerably.

Reply:
Adding the value to the gae gives you the return. See the docstring there